### ISSUE TITLE
can't visit the directory which created by ln -s command

### ISSUE THREAD

**Issue by Allsochen** [billziss-gh/sshfs-win#154]
can't visit the directory which created by ln -s command

**Comment by billziss-gh** [comment]
The `ln -s` command creates symbolic links and not directories. I am assuming that you created a symlink to a directory on a UNIX system (not Cygwin) using `ln -s`.

If the symlink points to a target outside the SSHFS file system, unfortunately this cannot be made to work.

**Comment by Brutus11111** [comment]
Can comfirm this bug.

Example:

ln -s /home/username/path/to/notes/folder/that/i/need/for/work/and/that/are/really/important /home/username/important
then on the Windows side, new fileshare to \\sshfs\user@ip\important

That doesn't work.

**Comment by phwelo** [comment]
Agreed, I use pretty heavy symlinking and get this error trying to go to the links.  
![image](https://user-images.githubusercontent.com/5237162/143727372-cf754d1a-349b-4fc6-a03e-df77ed529949.png)

**Comment by billziss-gh** [comment]
There were a number of symlink issues that were resolved in WinFsp 2022 Beta4. Please make sure to try with that version.

If you are running SSHFS-Win from the command line please make sure to always use the `-orellinks` option as well.

**Comment by dajuric** [comment]
The bug is still present in WinFsp 2023 version.
